autoload/ale/fixers/hclfmt.vim

```diff
@@ -0,0 +1,15 @@
+" Author: Peter Benjamin <petermbenjamin@kourouma.me>
+" Description: Integration of hclfmt with ALE.
+
+call ale#Set('hcl_hclfmt_executable', 'hclfmt')
```

Comment:
Add this line to the linter file too, and use the same executable variable for the linter. Add documentation for the option.